openstack/identity/v3/extensions/tokens/requests.go

```diff
@@ -114,69 +86,10 @@ func Create(c *gophercloud.ServiceClient, options extensions.AuthOptions, scope
 			}
 		} else {
 			// If no password or token ID are available, authentication can't continue.
-			return createErr(ErrMissingPassword)
+			return createErr(tokens.ErrMissingPassword)
 		}
 	} else {
-		// Password authentication.
```

Comment:
Why did you remove password authentication? We can use this if we provide trustID.